<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExplorerBar</class>
 <widget class="QWidget" name="ExplorerBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Explorer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
border: none;
}

QToolButton:hover, QToolButton:on  {
border: 1px solid rgb(200,200,200);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="leFilter">
       <property name="statusTip">
        <string>List of name filters to apply to the files (separated by ; without spaces)</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
       <property name="text" stdset="0">
        <string>*.*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbClearFilter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Clear name filter</string>
       </property>
       <property name="statusTip">
        <string>Disable name filtes and show all the files</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tbSynchReq">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Jump to current file</string>
       </property>
       <property name="statusTip">
        <string>Point to the current file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/view-sidetree.png</normaloff>:/images/view-sidetree.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbSynch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Synchronize with tabs</string>
       </property>
       <property name="statusTip">
        <string>Automatically change the path when current tab has been changed</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/synch.png</normaloff>:/images/synch.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="leDir">
       <property name="statusTip">
        <string>Directory to jump to</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbRoot">
       <property name="toolTip">
        <string>Go root</string>
       </property>
       <property name="statusTip">
        <string>Navigate to the root directory (Ctrl+Home)</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/browse-root.png</normaloff>:/images/browse-root.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbUp">
       <property name="toolTip">
        <string>Go up</string>
       </property>
       <property name="statusTip">
        <string>Navigate one directory up (Backspace)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/browse-up.png</normaloff>:/images/browse-up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbPickItem">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Jump to folder</string>
       </property>
       <property name="statusTip">
        <string>Pick the folder to jump to</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="tvDirs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="rootIsDecorated" stdset="0">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights" stdset="0">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionCopyPath">
   <property name="text">
    <string>Copy path</string>
   </property>
   <property name="toolTip">
    <string>Copy current item's path to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionCopyName">
   <property name="text">
    <string>Copy name</string>
   </property>
   <property name="toolTip">
    <string>Copy current item's name to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open current item</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbClearFilter</sender>
   <signal>clicked()</signal>
   <receiver>leFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
